assets/src/components/propertiesList.tsx

```diff
@@ -90,14 +109,17 @@ const highlightRegex = (highlightText: string): RegExp => {
   return new RegExp(allowNonAlphanumeric, "i")
 }
 
+const labelName = (label: string): string =>
+  label.toLowerCase().replace(" ", "-")
```

Comment:
I'm a little hesitant about using human-facing copy as css property names. Could we instead have a separate (maybe optional) field in `Property` for the css modifier?